cypress/fixtures/success.json

```diff
@@ -0,0 +1,3 @@
+{
+  "success": "A confirmation email for state election reminders has been sent to "
+}
```

Comment:
Once our BE is refactored to return this statement, will this fixture need to be refactored as well to include the email address that was submitted?